### DISCUSSION TITLE
Database error on new installation

### DISCUSSION THREAD

**Discussion by pa3hfj** [magicbug/Cloudlog#2471] [Q&A] [answered]
Hello Cloudlog users,

I am trying to install a new Cloudlog logbook on my server, but after filling in the details on the installation screen the following database error occurs:

```
A Database Error Occurred

Error Number: 1293

Incorrect table definition; there can be only one TIMESTAMP column with CURRENT_TIMESTAMP in DEFAULT or ON UPDATE clause

ALTER TABLE `api` ADD last_used TIMESTAMP DEFAULT NOW() NOT NULL AFTER `last_change`

Filename: migrations/116_add_timestamp_to_api.php

Line Number: 13
```
I could not find any pointers to resolve this problem, hopefully someone can get me on the right track. In the past I have successfully installed several version 1 logbooks on the same server, this is my first try to install a version 2 type.

_OS: Nethserver 7.9, based on CentOS 7
Database: MySQL (mariaDB 5.5.68)
PHP: 8.0
Type of installation: git clone (cloudlog version 2.4.8)_

With regards,
Wijnand PA3HFJ.

**Comment by AndreasK79** [comment, marked as answer]
Hi,

Seems to me that your MariaDB version is too old:
https://stackoverflow.com/questions/4489548/why-there-can-be-only-one-timestamp-column-with-current-timestamp-in-default-cla

Give it a go with a newer version and see if that helps.

**Reply by pa3hfj** [reply]
Thanks for your answer! Because of other applications running on the server I choose not to change anything on it for the time being but to wait for the new Nethserver 8 being production-ready so I can dump the no longer maintained CentOS. Instead I spun up a new VM based on Ubuntu server with more up-to-date software versions. On this server I could install cloudlog with no problems. So again a happy cloudlog user here!

73, Wijnand PA3HFJ.
